File: 058.Length of Last Word/Sidney/LengthofLastWord.c
int lengthOfLastWord(char* s) {
    int len = 0;
    for(int i = strlen(s)-1; i >= 0; i--) {
        if(s[i] == ' ') {
            if(len==0) {
                continue;
            }
            else {
                break;
            }
        }
        else {
            len++;
        }
    }
    return len;
}

File: 058.LengthOfLastWord/Ddfish/LengthOfLastWord.swift
class Solution {
    func lengthOfLastWord(_ s: String) -> Int {
    
        let subStrArray = s.components(separatedBy: " ")
        for str in subStrArray.reversed() {
            if(str != ""){
                return str.characters.count
            }
        }
        return 0
    }
}
